src/rules/noUnusedVariableRule.ts

```diff
@@ -193,7 +193,13 @@ class NoUnusedVariablesWalker extends Lint.RuleWalker {
             }
         }
 
+        // skip checking parameter declarations for abstract methods
+        // They can't have a body so parameters are always unused
```

Comment:
let's just simplify to a one-line comment (this will fix the lint failure as well)

```ts
// abstract methods can't have a body so their parameters are always unused
```